test/Microsoft.AspNetCore.Session.Tests/SessionTests.cs

```diff
@@ -105,8 +105,11 @@ public class SessionTests
                {
                    app.UseSession(new SessionOptions
```

Comment:
Yeah, I just meant a cleanup pass for all middleware doing this, not something for this PR, I'll file an issue